src/ManageCourses.Api/Data/NullAccessRequestEmailService.cs

```diff
@@ -0,0 +1,12 @@
+﻿using GovUk.Education.ManageCourses.Domain.Models;
+
+namespace GovUk.Education.ManageCourses.Api.Data
+{
+    public class NullAccessRequestEmailService : IAccessRequestEmailService
```

Comment:
It's not for debugging, it's for config resolution: One config means you can (and should) send emails --> AccessRequestEmailService. The other config means you can't (and shouldn't). Factory pattern for strategy selection's pretty standard, not sure why it would be overkill?